mito-ai/src/Extensions/AiChat/ChatMessage/ChatInput.tsx

```diff
@@ -40,8 +40,8 @@ const ChatInput: React.FC<ChatInputProps> = ({
 
         textarea.style.minHeight = 'auto';
         textarea.style.height = !textarea.value || resetHeight
-            ? '40px' 
-            : `${Math.max(40, textarea.scrollHeight)}px`;
+            ? '80px' 
+            : `${Math.max(80, textarea.scrollHeight)}px`;
```

Comment:
Doubling the size here. At 40px, it was roughly one line long. Most users will have a multi-line question.